components/brave_webtorrent/extension/manifest.json

```diff
@@ -20,6 +20,6 @@
       "listen": "*:*"
     }
   },
-  "content_security_policy": "default-src 'self'; connect-src 'self'; font-src 'self' data:; script-src 'self'; media-src 'self' http://127.0.0.1:*; form-action 'none'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; frame-src 'self' http://127.0.0.1:*;",
```

Comment:
cc @diracdeltas for security review

Reply:
@yrliou could you explain why this change is needed? it isn't in the browser-laptop version of webtorrent so i wonder if we can work around it.

Reply:
When we click start torrent, the background page of webtorrent component extension will initiate a request to fetch the actual .torrent, for example, from https://webtorrent.io/torrents/big-buck-bunny.torrent
Without this change, I'm getting `"Refused to connect to 'https://webtorrent.io/torrents/big-buck-bunny.torrent' because it violates the following Content Security Policy directive: "connect-src 'self'".` in the background page.
I think for the browser-laptop case, these things are happened in browser process instead of the extension.

Reply:
@yrliou thanks for the explanation. Have you tried seeing if this works with an HTTP (not HTTPS) endpoint? I'm afraid Chromium might block that for being mixed content even if it is whitelisted in CSP.

If so we should take another approach like downloading the .torrent file to a tempfile (which would remove the need to whitelist http/https connections in the CSP)

Reply:
@diracdeltas I just tried fetching things from HTTP endpoint in the background page using simple-get(https://github.com/feross/simple-get), which is the same as how webtorrent libs make those requests.
Adding http: in CSP did allow us to fetch things normally.

Reply:
sgtm

Reply:
Tried with http://www.slackware.com/torrents/slackware-14.2-install-d1.torrent, .torrent file are successfully fetched.